kong/plugins/proxy-cache/handler.lua

```diff
@@ -173,11 +176,25 @@ local function cacheable_response(conf, cc)
       return false
     end
 
+    local opts = { ignorecase_params = { "charset" } }
```

Comment:
I think we should remove `ignorecase_params` from the mimetype utility. See my comment on syster pr.